guides/source/caching_with_rails.md

```diff
@@ -286,90 +349,174 @@ simply be explicit in a comment, like:
 <%= some_helper_method(person) %>
 ```
 
-### Low-Level Caching
+Solid Cache
+-----------
 
-Sometimes you need to cache a particular value or query result instead of caching view fragments. Rails' caching mechanism works great for storing any serializable information.
+Solid Cache is a database-backed Active Support cache store and is a great choice for applications that need to cache large amounts of data.
 
-The most efficient way to implement low-level caching is using the `Rails.cache.fetch` method. This method does both reading and writing to the cache. When passed only a single argument, the key is fetched and value from the cache is returned. If a block is passed, that block will be executed in the event of a cache miss. The return value of the block will be written to the cache under the given cache key, and that return value will be returned. In case of cache hit, the cached value will be returned without executing the block.
+By leveraging the speed of modern SSDs, like NVMe, Solid Cache is designed to overcome the limitations of traditional memory-only caches like Redis and Memcached. You get the following benefits with With Solid Cache:
 
-Consider the following example. An application has a `Product` model with an instance method that looks up the product's price on a competing website. The data returned by this method would be perfect for low-level caching:
+- Larger Cache Sizes: The cached data is stored on a disk instead of in memory - this allows you to cache large amounts of data without worrying about the high cost. Memory-only caches like Redis and Memcached are expensive to scale because the come with very little storage capacity.
 
-```ruby
-class Product < ApplicationRecord
-  def competing_price
-    Rails.cache.fetch("#{cache_key_with_version}/competing_price", expires_in: 12.hours) do
-      Competitor::API.find_price(id)
-    end
-  end
-end
+- Cost-Effective: Solid Cache is much more cost-effective than traditional memory-only caches. Memory-only caches are expensive to scale because they require a lot of memory. Solid Cache is much cheaper because it uses disk storage which is much cheaper than memory.
+
+- Simplified Infrastructure: No need for additional dependencies like Redis or Memcached. Additionally, Solid Cache uses a relational database to store cached data, making it simpler to integrate into existing database setups.
```

Comment:
This *might* be overselling it a little bit :) In practice, if your application is making heavy use of caching you would have a dedicated database for caches (and not use your primary database) so that would need to be added to your infrastructure. In the cases where the application is not self hosted and is in "the cloud", it's typically cheaper to get a Redis instance than another database instance. And Redis is dead simple, just a key/value store. So we might contradict the point :)

Overall, my reading up on Solid Cache so far has revealed that there are various tradeoffs and ultimately you would have to test it out with your application performance, it's not a "everyone should use this!" technology. But I haven't given it a spin in production so 🤷‍♀️

Reply:
Thanks for dropping your thoughts here! I ended up removing these points and just added a brief sentence "By leveraging the speed of modern SSDs like NVMe, Solid Cache overcomes the limitations of traditional memory-only caches, offering cost-effective caching with larger storage capacity and simplified infrastructure.".